tracing-appender/src/rolling.rs

```diff
@@ -37,6 +37,19 @@ impl io::Write for RollingFileAppender {
     }
 }
 
+/// A function which creates a hourly rolling file appender. This appender implements only the
+/// `Write` trait and can be used with `NonBlocking` to provide a `MakeWriter` to be used with a
+/// tracing subscriber.
+/// # Examples
+/// ``` rust,ignore
+/// let appender = rolling::hourly("/some/path", "rolling.log");
+/// let (non_blocking_appender, _guard) = non_blocking(appender);
+///
+/// let subscriber = tracing_subscriber::fmt().with_writer(non_blocking_appender);
+///
+/// tracing::subscriber::with_default(subscriber.finish(), || {
+///     tracing::event!(tracing::Level::INFO, "Hello");
+/// });
```

Comment:
```suggestion
/// Creates an hourly, rolling file appender.
///
/// The appender returned by `rolling::hourly` can be used with `non_blocking` to create
/// an non-blocking, hourly file appender.
/// 
/// # Examples
/// 
/// ```no_run
/// let appender = rolling::hourly("/some/path", "rolling.log");
/// let (non_blocking_appender, _guard) = non_blocking(appender);
///
/// let subscriber = tracing_subscriber::fmt().with_writer(non_blocking_appender);
///
/// tracing::subscriber::with_default(subscriber.finish(), || {
///     tracing::event!(tracing::Level::INFO, "Hello");
/// });
```

Reply:
I took the wording suggestion from you but kept the suggestions to wrap the example in fn main and fn doc